### PR TITLE
Fixes #1091: Added Tools Tab below screen width 496px

### LINKED ISSUE #1091: Missing Tools Tab on screen width less than 496
**I'm submitting a ...**
- [X] bug report
- [ ] feature request

**Current behavior:**

Even though there is enough space to show Tools Tab below screen width less than 496px, it is still not present.

**Expected behavior:**

It should be present with proper spacing.

**Screenshots of the issue:**

screenshot from 2018-07-10 23-57-02

### PR DESCRIPTION
**Pull request by praveenojha33**
Fixes #1091 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)
- [x] Added Surge preview link

#### Changes proposed in this pull request:
Added Tools Tab below screen width 496px

- https://pr-1092-fossasia-susper.surge.sh

image

### CODE REVIEW

File: src/app/results/results.component.css [hunk 1]
```diff
@@ -418,27 +418,38 @@ div.autocorrect {
...
+  }
+
+  @media screen and (max-width: 327px) {
+    #search-options li {
+      padding: 0 1vw 12px!important;
+    }
+    }
+@media screen and (max-width: 283px) {
+    #search-options li {
+      padding: 0 0vw 12px!important;
```
Comment (simsausaurabh) [inline src/app/results/results.component.css]: Needs to be made more appropriate and codacy fails for it also.

File: src/app/results/results.component.css [hunk 2]
```diff
@@ -709,7 +720,7 @@ div.autocorrect {
   }
 
   #search-options li {
-    padding: 0 4vw 12px;
+    padding: 0 3vw 12px;
```
Comment (simsausaurabh) [inline src/app/results/results.component.css]: Either use ```vw``` or ```px```

File: src/app/results/results.component.css (hunk 1 above)
Comment (praveenojha33) [reply src/app/results/results.component.css]: Codacy is complaining about using important which is already used at many places. There is no problem with that.

File: src/app/results/results.component.css (hunk 2 above)
Comment (praveenojha33) [reply src/app/results/results.component.css]: px is used for fixed space at bottom and vw is used for relative spacing at right between the tab.

File: src/app/results/results.component.css (hunk 1 above)
Comment (simsausaurabh) [reply src/app/results/results.component.css]: @praveenojha33 I think Codacy states not to use it. And use same units for any single property CSS.

Comment (praveenojha33) [reply src/app/results/results.component.css]: `important` is already used at many places in project.

File: src/app/results/results.component.css (hunk 2 above)
Comment (simsausaurabh) [reply src/app/results/results.component.css]: px can do it same?

Comment (praveenojha33) [reply src/app/results/results.component.css]: We want fixed padding at bottom and relative padding at right.

File: src/app/results/results.component.css (hunk 1 above)
Comment (simsausaurabh) [reply src/app/results/results.component.css]: Could you fix Codacy?

File: src/app/results/results.component.css (hunk 2 above)
Comment (simsausaurabh) [reply src/app/results/results.component.css]: Could you fix Codacy?

File: src/app/results/results.component.css (hunk 1 above)
Comment (praveenojha33) [reply src/app/results/results.component.css]: I will have to remove `important` but it will take more time and effort for no reason.

**Comment by simsausaurabh** [comment]
@praveenojha33 Please look into these:

screenshot from 2018-07-11 16-22-31

screenshot from 2018-07-11 16-22-42

screenshot from 2018-07-11 16-22-50

**Comment by praveenojha33** [comment]
@simsausaurabh I have made changes please review again.

**Comment by praveenojha33** [comment]
@mariobehling I have made changes. Please see if it is fine 
image

**Comment by mariobehling** [comment]
Still not working as expected. Please read up how to keep the same ratio of a picture. Don't squeeze images, don't stretch them. Check with co-developers and read up on it, please.

**Pull request merged**